payroll/models/hr_payslip.py

```diff
@@ -339,9 +339,18 @@ def get_worked_day_lines(self, contracts, date_from, date_to):
         for contract in contracts.filtered(
             lambda contract: contract.resource_calendar_id
         ):
+
+            # Adds support for the hr_public_holidays module.
+            # This adds support to exclude public holidays from work days computation.
+            # If you don't use the module, it don't make any difference or harm.
+            contract = contract.with_context(
+                employee_id=self.employee_id.id, exclude_public_holidays=True
+            )
+
             day_from = datetime.combine(date_from, time.min)
             day_to = datetime.combine(date_to, time.max)
             day_contract_start = datetime.combine(contract.date_start, time.min)
+
```

Comment:
Don't add empty lines inside a method

Reply:
And the same for the rest of empty lines that you introduce in this PR

Reply:
The empty lines are there from previous versions of the module, I think the person who wrote it want it to separate different parts of the method. If you like i Can refactor the method and we fix that in this PR.

Reply:
You are adding 3 empty lines in this PR . Please remove that ones.